user_management/api/views.py

```diff
@@ -188,19 +240,33 @@ def get_object(self):
 
 
 class UserList(generics.ListCreateAPIView):
+    """
+    User list and create view.
+
+    Allow to `GET` a list users and to `POST` new user for admin user only.
+    """
     model = User
     permission_classes = (IsAuthenticated, permissions.IsAdminOrReadOnly)
     serializer_class = serializers.UserSerializerCreate
 
 
 class UserDetail(generics.RetrieveUpdateDestroyAPIView):
+    """
+    User detail view.
```

Comment:
Display information about a user.

Allow admin users to update or delete user information.